modules/ROOT/pages/edifact-send-settings.adoc

```diff
@@ -13,10 +13,10 @@ To configure a partner's EDIFACT send settings:
 * <<character-set,Character set & delimiters>>
 . Click *Save*.
 
-The values you configure in the partner's settings page are used by default for all EDIFACT messages sent to that trading partner. You can override the settings you configure at the message flow level by selecting the custom option while creating an outbound message flow.
+The values you configure in the partner's settings page are used by default for all EDIFACT messages sent to that partner. You can override the settings you configure at the message flow level by selecting the custom option while creating an outbound message flow.
 
 [NOTE]
-It can take up to five minutes for your configuration changes to sync with Mule.
+It can take up to five minutes for changes to take effect on the runtime.
```

Comment:
See previous similar comment. 

(I try to avoid "It can" and other "be word" construction.)

Reply:
OK